app/react/Forms/components/specs/__snapshots__/DateRange.spec.js.snap

```diff
@@ -5,11 +5,15 @@ exports[`DateRange should allow using the local timezone 1`] = `
   <div
     className="DatePicker__From"
   >
-    <Connect(Translate)
-      translationKey="Label date \\"From\\""
+    <span
```

Comment:
Did prettier really asked for 3 lines instead of a single line of a simple span with a single className?